terraform/modules/gke/main.tf

```diff
@@ -72,11 +74,52 @@ module "gke_cluster" {
   }
 }
 
+# TODO: Define GKE node pool separate from the cluster
```

Comment:
If this is not used anywhere in the gke module now, can you remove it?

Reply:
Done